pkg/kubehound/storage/graphdb/janusgraph_edge_writer.go

```diff
@@ -86,31 +90,25 @@ func (jgv *JanusGraphEdgeWriter) startBackgroundWriter(ctx context.Context) {
 // batchWrite will write a batch of entries into the graph DB and block until the write completes.
 // Callers are responsible for doing an Add(1) to the writingInFlight wait group to ensure proper synchronization.
 func (jgv *JanusGraphEdgeWriter) batchWrite(ctx context.Context, data []any) error {
-	span, ctx := tracer.StartSpanFromContext(ctx, telemetry.SpanJanusGraphOperationBatchWrite, tracer.Measured())
+	span, ctx := tracer.StartSpanFromContext(ctx, telemetry.SpanJanusGraphOperationBatchWrite,
+		tracer.Measured(), tracer.ServiceName(TracerServicename))
 	span.SetTag(telemetry.TagKeyLabel, jgv.builder)
 	defer span.Finish()
 	defer jgv.writingInFlight.Done()
 
-	tx := jgv.traversalSource.Tx()
```

Comment:
I know we've gone round in circles on this but as per ALQ's comment we should not ever be using transactions here because it slows stuff for no benefit. I think it was papering over another issue that we have since fixed so no plan to bring these back (even with other backends - transactions still happen, but managed by the backend itself)